AssetRipper.GUI/ConsoleOptions.cs

```diff
@@ -0,0 +1,350 @@
+				option.AddAlias("-i");
+				option.IsRequired = true;
+				option.Arity = ArgumentArity.OneOrMore;
+
+				option.AddValidator((symbolResult) =>
+				{
+					List<string>? inputs = symbolResult.GetValueForOption(option);
+
+					if (inputs == null)
+						return;
+
+					foreach (string input in inputs)
+					{
+						string path = ExecutingDirectory.Combine(input);
+
+						if (!MultiFileStream.Exists(path) && !Directory.Exists(path))
+						{
+							symbolResult.ErrorMessage = MultiFileStream.IsMultiFile(path)
+								? $"File '{path}' doesn't have all parts for combining"
+								: $"Neither file nor directory with path '{path}' exists";
+
+							Environment.ExitCode = (int)ExitCode.ArgumentError;
+							return;
+						}
+					}
+				});
+
+				_inputOption = option;
+
+				return option;
+			}
+		}
+
+		private static Option<DirectoryInfo>? _outputOption;
+
+		public static Option<DirectoryInfo> outputOption
+		{
+			get
+			{
+				if (_outputOption != null)
+					return _outputOption;
+
+				Option<DirectoryInfo> option = new Option<DirectoryInfo>(name: "--output", description: "Directory to export to (will be cleared if already exists)");
+				option.AddAlias("-o");
+				option.SetDefaultValue(DefaultOutputPath);
+				option.IsRequired = false;
+
+				option.AddValidator((symbolResult) =>
+				{
+					DirectoryInfo? dirInfo = symbolResult.GetValueForOption(option);
+
+					if (dirInfo == null)
+						dirInfo = new DirectoryInfo(DefaultOutputPath);
+
+					string path = ExecutingDirectory.Combine(dirInfo.FullName);
+
+					if (!Directory.Exists(Path.GetDirectoryName(path)))
+					{
+						symbolResult.ErrorMessage = "Directory for output does not exist";
+						Environment.ExitCode = (int)ExitCode.ArgumentError;
+					}
+				});
+
+				_outputOption = option;
+
+				return option;
+			}
+		}
+
+		private static Option<bool>? _quitOption;
+
+		public static Option<bool> quitOption
+		{
+			get
+			{
+				if (_quitOption != null)
+					return _quitOption;
+
+				Option<bool> option = new Option<bool>(name: "--quit", description: "Quit after ripping");
+				option.AddAlias("-q");
+				option.SetDefaultValue(DefaultQuit);
+
+				_quitOption = option;
+
+				return option;
+			}
+		}
+
+		private static Option<bool>? _verboseOption;
+
+		public static Option<bool> verboseOption
+		{
+			get
+			{
+				if (_verboseOption != null)
+					return _verboseOption;
+
+				Option<bool> option = new Option<bool>(name: "--verbose", description: "Verbose logging output");
+				option.AddAlias("-v");
+				option.SetDefaultValue(DefaultVerbose);
+
+				_verboseOption = option;
+
+				return option;
+			}
+		}
+
+		private static Option<FileInfo>? _logFileOption;
+
+		public static Option<FileInfo> logFileOption
+		{
+			get
+			{
+				if (_logFileOption != null)
+					return _logFileOption;
+
+				Option<FileInfo> option = new Option<FileInfo>(name: "--log-file", description: "File to log to");
+				option.AddAlias("-w");
```

Comment:
Why is `-w` an alias for this?

Reply:
Stands for `write`, since `-l` was already taken. Which letter would you suggest?